webapp/src/main/java/life/qbic/usermanagement/registration/RegistrationMessageFactory.java

```diff
@@ -0,0 +1,20 @@
+package life.qbic.usermanagement.registration;
+
+/**
+ * Creates registration messages.
+ */
+public class RegistrationMessageFactory {
+
+  public static String registrationMessage(String fullName) {
```

Comment:
Since we already know, that we will provide a URL to confirm the email address, we can add it as a method parameter.

Can be a String, must not be a URL, since it is formatting only.

Reply:
I would extend it once we have it in place. Atm, we do not have an URL so I would opt for a functioning version where a simple email is sent without any action required from the user.